docker/etc/pki/yarnpkg.gpg.key

```diff
@@ -1,4 +1,5 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v2
```

Comment:
This makes me think they maybe just upgraded GPG or something

Reply:
That's what I thought too. The key is a lot longer but I forget if public keys have consistent lengths because I totally forget how GPG's encryption works 😅